xml/admin_gui_dashboard.xml

```diff
@@ -2287,6 +2287,92 @@ Oct 04 07:48:53 doc-ses6min4 systemd[1]: Started Ceph rbd mirror daemon.
+     <para>
+       &openssl; is an open-source command line tool that is commonly used
+       to generate private keys, create Certificate Signing Requests (CSR), install your SSL/TLS
+       certificate, and identify certificate information. The following
+       instructions illustrate to generate a self-signed or trusted third-party
+       certificate using &openssl;:
+     </para>
+<!-- Source: https://www.suse.com/support/kb/doc/?id=000019241 -->
+     <procedure>
+       <step>
+         <para>
+           Generate a Private Key:
+         </para>
+<screen>
+&prompt.cephadm;openssl genrsa -des3 -out server.key 2048
+</screen>
+         <para>
+           Type the pass phrase to protect the key.
+         </para>
+       </step>
+       <step>
+         <para>
+           Generate a CSR:
+         </para>
+<screen>
+&prompt.cephadm;openssl req -new -key server.key -out server.csr
+</screen>
+         <para>
+           Enter the passphrase, and fill in the <literal>Country Name</literal>,
+           <literal>State or Province Name</literal>, <literal>Locality Name</literal>, <literal>Organization Name</literal>,
+           <literal>Organizational Unit Name</literal>, <literal>Common Name</literal>,
+           <literal>Email Address</literal>.
+         </para>
+         <note>
+           <para>
+             The <literal>Common Name</literal> should be the FQDN name of the
+             server. For example, <literal>server.mydomain.com</literal>.
+           </para>
+         </note>
+         <para>
+	   When asked for a <literal>challenge</literal> password and optional company name,
+           leave it blank.
+         </para>
+       </step>
+       <step>
+         <para>
+           To sign the certificate, select from the following options:
+         </para>
+         <itemizedlist>
+           <listitem>
+             <formalpara>
+               <title>Trusted 3rd-party Certificate Authority</title>
+               <para>Send the
+               CSR to the third party for their
+               signing. The following files should be received: Server certificate (public key)
+               and the Intermediate CA and the bundles that chain to the Trusted Root CA.
+               </para>
+             </formalpara>
+           </listitem>
+           <listitem>
+             <formalpara>
+               <title>Self-signed</title>
+               <para>Sign the certificate with &openssl;:</para>
+             </formalpara>
+<screen>
+openssl x509 -req <emphasis role="strong">-days 730</emphasis> -in server.csr -signkey server.key -out server.crt
+</screen>
+             <para>
+               Increase or decrease 730 as needed. This is the number
+               of days the certificate is valid for.
+             </para>
+           </listitem>
+         </itemizedlist>
+       </step>
+       <step performance="optional">
+         <para>
+           If needed, create a concatenated PEM file:
```

Comment:
What does PEM stand for?

Reply:
Privacy Enhanced Mail - whilst I agree that all acronyms should be expanded, this is a very well known one and people would not refer to it as a "Privacy Enhanced Mail" file, only PEM.